### ISSUE TITLE
3d transforms: moving a transform up in the list loses focus

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#206]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/3dtransforms/

Steps:

Tab to the up arrow button for the second transform (rotateY)
Press enter to activate it

Expected:

Transform to move to first in the list, focus to still be on the same button.

Actual:

Transform moves up but focus is lost (no idea where it goes
Note than moving a transform down works as expected. 

The X button has the same issue, but not 100% sure what should have focus once it is deleted.

**Comment by alrra** [comment]
@dstorey @sarvaje This demo is external, namely, it's from https://testdrive-archive.azurewebsites.net/Graphics/hands-on-css3/hands-on_3d-transforms.htm.

**Comment by molant** [comment]
@dstorey should we just take down this one and build it again under the new section that @melanierichards proposed with others?

**Comment by melanierichards** [comment]
@molant I'd support that decision cc @dstorey @alrra

**Comment by ststimac** [comment]
I second @molant's proposal. This was on my list of demos to redesign and there are a ton of sub-parts to this demo: 
<img width="1260" alt="screen shot 2016-12-16 at 2 32 19 pm" src="https://cloud.githubusercontent.com/assets/18073131/21280721/8379ac86-c39c-11e6-8437-3ca48528f096.png">

And I think it would be most time efficient to build from scratch if possible.

**Comment by melanierichards** [comment]
Ok so it seems like people are on board with taking it down. Is this one we can take down now, or do we need to have the replacement on the live site before we can do so?

**Comment by molant** [comment]
We can do it from the administration panel. We should probably add a redirect to somewhere (testdrive?).
@sarvaje can do this

**Comment by sarvaje** [comment]
do you want to do it now?
